src/test/kotlin/no/nav/familie/tilbake/LauncherLocal.kt

```diff
@@ -12,7 +11,6 @@ class LauncherLocal
 fun main(args: Array<String>) {
 
     SpringApplicationBuilder(ApplicationConfig::class.java)
-            .initializers(DbContainerInitializer())
```

Comment:
Dette er vel her docker-basen blir initialisert. Vi trenger vel den?

Reply:
> Omstrukturering av pakker burde vært en egen PR. Dette blir en fryktelig tung merge-jobb.
> Domene-endringen burde også vært en egen PR.
> Hva skjedde med å opprette draft-request for å ikke kjøre db-oppdateringer før de er godkjent?

I dag finnes det bare 2 PR. En for database som du opprettet og denne. Vi(alle utviklere i tilbakekreving team) hadde et diskusjon om database PR og bestemte oss å ikke sjekke inn det. I stedet skal vi bruke den når vi trenger det. F.eks. når man jobber med fakta, kan kopiere nødvendige tabell og domene klasser fra denne PR-en. Derfor endret vi database PR-en til draft og holde den PR. Så vi trenger ikke å dytte disse omstrukturering i en annen PR.

Reply:
Jeg er ikke sikkert på det. Men det kan vi fikse senere hvis det ikke funker

Reply:
Det funker ikke.